net-nds/389-ds-base/389-ds-base-1.4.2.15.ebuild

```diff
@@ -1,39 +1,40 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-WANT_AUTOMAKE="1.13"
+WANT_AUTOMAKE="1.16"
 
-inherit user eutils multilib flag-o-matic autotools
-
-DESCRIPTION="389 Directory Server (core librares and daemons )"
-HOMEPAGE="http://www.port389.org/"
-SRC_URI="http://www.port389.org/sources/${P}.tar.bz2"
+inherit eutils multilib flag-o-matic autotools systemd db-use
```

Comment:
is eutils still needed?

Reply:
> 
> 
> i had hoped someone would pick this up!
> 
> you chose 1.4.2 branch for replication purposes i assume?
> looks like this needs slotting which sounds kinda scary to me, then again, fedora just does update this too without keeping the old version...
> 
> i haven't tested this but i have a few remarks

I'm not quite sure it needs slotting as it's not like a library that installs different versions - as such I haven't bothered with slotting. 

In any case the 1.4.4 branch has been released now so I can remain up to date and track that branch minor version for now while I look into it some more.

Reply:
it was only a general thought but i think only postgresql does slotting right with a lot of work so its probably not gonna happen.

also there is the thought to be had if we really *want* to care, because technically 3 branches would have to be maintained ( +1 for RHDS sometimes ) but thats only for replication and also only replication actually breaks with slaves being *newer* then the master, i haven't looked into that yet

Reply:
> 
> 
> it was only a general thought but i think only postgresql does slotting right with a lot of work so its probably not gonna happen.
> 
> also there is the thought to be had if we really _want_ to care, because technically 3 branches would have to be maintained ( +1 for RHDS sometimes ) but thats only for replication and also only replication actually breaks with slaves being _newer_ then the master, i haven't looked into that yet

That's a good point - my view is if someone needs a specific minor branch they can use package.mask to mask versions greater than the .0 release of the next highest major branch and manage it that way - slotting would only work if you could have multiple versions installed and I don't think that's supported by upstream so would be a lot of work for very little gain.